### PR TITLE
fix(blind-spots): address review findings

### PR DESCRIPTION
**Pull request by joeharris76**
Follow-up to PR #56 addressing the review findings before relying on blind-spot capture as review infrastructure.

Changes:
- Block path traversal and invalid IDs in `sweep_blind_spots.py`.
- Normalize CRLF frontmatter and mixed YAML date scalar handling.
- Type-check enum fields so nested YAML fails with validation errors instead of tracebacks.
- Preserve unrelated frontmatter formatting when triaging.
- Insert triage-log entries into the `## Triage log` section rather than appending blindly to EOF.
- Reject invalid triage option combinations.
- Document filename collision handling and protocol drift signals.
- Add focused fast unit coverage for validator and sweep edge cases.

Validation run locally in a fresh clone:
- `uv run -- python -m pytest tests/unit/scripts/test_blind_spot_tools.py -q -n 0`
- `uv run --project _project/scripts -- python _project/scripts/validate_blind_spot.py --all`
- `uv run -- python -m ruff check _project/scripts/validate_blind_spot.py _project/scripts/sweep_blind_spots.py tests/unit/scripts/test_blind_spot_tools.py`
- `uv run -- python -m ruff format --check _project/scripts/validate_blind_spot.py _project/scripts/sweep_blind_spots.py tests/unit/scripts/test_blind_spot_tools.py`

### CODE REVIEW

File: _project/scripts/sweep_blind_spots.py [hunk 1]
```diff
@@ -188,43 +212,72 @@ def cmd_report(bs_dir: Path, args: argparse.Namespace) -> int:
...
+    if "\n" in rendered:
+        raise ValueError(f"{field} must render as a single YAML line")
+    return rendered
+
+
+def replace_frontmatter_field(raw: str, field: str, value: str) -> str:
+    line = render_scalar_field(field, value)
+    pattern = re.compile(rf"(?m)^{re.escape(field)}:.*$")
+    if pattern.search(raw):
+        return pattern.sub(line, raw, count=1)
```
[the remark this answers is not in the document]
Comment (joeharris76) [reply _project/scripts/sweep_blind_spots.py:227]: Follow-up sweep actioned this Codex review comment.

        Disposition: no-current-action
        Branch: `fix/codex-followups-sweep-2026-05-04`

        Summary:
        Disposition: no-current-action

Evidence: Current [_project/scripts/sweep_blind_spots.py](/Users/joe/Developer/BenchBox.pool-04/_project/scripts/sweep_blind_spots.py:223) now replaces the full top-level field block, not just the matching line. Focused coverage already exists in [tests/unit/scripts/test_blind_spot_tools.py](/Users/joe/Developer/BenchBox.pool-04/tests/unit/scripts/test_blind_spot_tools.py:177), and the current audit records this PR #59 thread as fixed. `git status --short` is clean.

Verification: Ran an ad hoc `uv run -- python` probe for block-shaped `status` and `todo_id`, then `uv run -- python -m pytest tests/unit/scripts/test_blind_spot_tools.py -q -n 0` -> `9 passed`. Initial pytest without `-n 0` was blocked by an existing BenchBox global test lock from another worktree.

        Future sweeps skip comments that already have this marker reply.
